src/jit-format/jit-format.cs

```diff
@@ -644,11 +693,11 @@ public static bool RunClangFormat(List<string> filenames, bool fix, bool verbose
 
                 if (verbose)
                 {
-                    Console.WriteLine("Running: clang-format {0} {1}", formatFix, filename);
+                    Console.WriteLine("Running: clang-format {0} -style=file {1}", formatFix, filename);
```

Comment:
No, it isn't. I'm not 100% sure it's necessary, though @pgavlin pointed it out on Friday. I've tried clang-format with and without -style=file, and I get the same results, so it's just a bit of paranoia